.github/workflows/static-analysis.yml

```diff
@@ -21,10 +22,17 @@ jobs:
         with:
           path: |
             **/node_modules
-            .yarn/cache
+            .yarn
           key: ${{ runner.os }}-yarn-cache-${{ hashFiles('./yarn.lock') }}
 
-      - run: yarn workspaces focus @hyperlane-xyz/core
+      - name: yarn-install
+        run: yarn install
+
+      - name: foundry-install
+        uses: onbjerg/foundry-toolchain@v1
```

Comment:
nit: can minmize the number of foundry installs like have it under the same jobs? We do it 8 times in CI now